### ISSUE TITLE
Theano warning: downsample module moved to pool module

### ISSUE THREAD

**Issue by ebattenberg** [Lasagne/Lasagne#605]
Just updated my Theano.  

Getting this now:
```
...projects/Theano/theano/tensor/signal/downsample.py:5: UserWarning: downsample module has been moved to the pool module.
  warnings.warn("downsample module has been moved to the pool module.")
```

No worries.  Just an interface update.

**Comment by eduardo4jesus** [comment]
I've had this same warning. Up to now, it is "just a warn".

**Comment by f0k** [comment]
This happens on `import theano` (because Theano itself auto-imports the old module). We should update the imports for the pooling layers and update the Theano version in `requirements.txt` and `__init__.py` accordingly, though. When we did that, we could install a `warnings.filterwarning` in `__init__.py` to silence the warning, but I think we shouldn't (users might combine Lasagne with vanilla Theano and should still be warned about this).

**Comment by benanne** [comment]
Agreed, I don't think we should filter this.

**Comment by f0k** [comment]
Was resolved by #644.
